tests/lib/devicetree/emul/src/main.c

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2022 Google Inc
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <device_extern.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/emul.h>
+#include <ztest.h>
+#include <ztest_assert.h>
+#include <ztest_test_new.h>
+
+#define TEST_GPIO DT_NODELABEL(test_gpio_0)
+
+ZTEST(emul, test_emul_dt_get)
+{
+	const struct emul *emul = emul_get_binding(DT_LABEL(TEST_GPIO));
+	const struct emul *emul_static = EMUL_DT_GET(TEST_GPIO);
```

Comment:
Can you make this `static` so we can verify that the macro resolved as expected?